signalfx/resource_signalfx_dashboard.go

```diff
@@ -439,6 +445,15 @@ func getPayloadDashboard(d *schema.ResourceData) (*dashboard.CreateUpdateDashboa
 		cudr.AuthorizedWriters.Users = users
 	}
 
+	if val, ok := d.GetOk("tags"); ok {
+		var tags []string
+		tfValues := val.([]interface{})
```

Comment:
Does the `schema.Set` work here? or is the an `schema` type that would be usable here? 

Noticing their other code isn't using the raw interface, I suspect for a reason to avoid typed nils and make type checking easier.

Reply:
`schema.Set` wasn't working as I cannot assert anything about the tags values in the test cases as the order is unknown and there is no existing testing functions that allow you to assert on the values inside a set.

A `schema.List` doesn't exist either unfortunately.

I've copied and adapted this part from [here](https://github.com/atlassian-forks/terraform-provider-signalfx/blob/main/signalfx/resource_signalfx_dashboard.go#L539)